InfernoRed/module-usbstick
# Feature/read write files

Adds support for mounting usb drives and reading/writing arbitrary json files

## To Test
1. load the postman collection and environment
2. start up the api
3. insert a usb drive
4. run the postman collection steps in order

Expected: a test file is written to the usb device
